hicommonwealth/commonwealth
# updated tos

## Link to Issue
Closes: #8387 

## Description of Changes
-Updates Terms of Service

## "How We Fixed It"

-changed the TOS text 
## Test Plan
- go to `/terms`
- confirm that you now see the text that is connected to the PR
